csrankings-m.csv

```diff
@@ -157,6 +157,7 @@ Mahshid Mojtabavi Naeini,Texas Tech University,https://www.depts.ttu.edu/provost
 Mai Anh Tien,Singapore Management University,https://sis.smu.edu.sg/faculty/profile/167371/MAI-Anh-Tien,5TgvaiwAAAAJ
 Mai ElSherief,Univ. of California - San Diego,https://mayelsherif.github.io/melsherif,Rykl2HkAAAAJ
 Mai Fadel,King Abdulaziz University,http://mfadel.kau.edu.sa/Default.aspx?Site_ID=0011432&Lng=EN,NOSCHOLARPAGE
+Mai Oudah,NYU Abu Dhabi,https://nyuad.nyu.edu/en/academics/divisions/science/faculty/mai-oudah.html,PUDnxbMAAAAJ

```

Comment:
Lecturer, not TT faculty; please remove.

Reply:
removed